src/blob_storage.cc

```diff
@@ -45,6 +45,11 @@ void BlobStorage::ExportBlobFiles(
 void BlobStorage::AddBlobFile(std::shared_ptr<BlobFileMeta>& file) {
   WriteLock wl(&mutex_);
   files_.emplace(std::make_pair(file->file_number(), file));
+  AddStats(stats_, cf_id_, TitanInternalStats::LIVE_BLOB_SIZE,
```

Comment:
Will all these properties be reconstructed on DB reopen?

Reply:
I think they are. During `Apply` in `VersionSet::Recover`.

Reply:
Hmm, it seems to me `VersionSet::Apply` doesn't reset `discardable_size`, so `live-blob-size` will be incorrect anyway. Let's address the issue separately.